auth/src/main/java/org/jboss/pnc/bacon/auth/model/CacheFile.java

```diff
@@ -80,11 +81,13 @@ public static void writeCredentialToCacheFile(
         }
     }
 
-    private static void createConfigFolderIfAbsent() {
-        if (!fileExists(Config.getConfigLocation())) {
+    private static void createConfigFolderIfAbsent() throws IOException {
+        Path path = Paths.get(Config.getConfigLocation());
+
+        if (!Files.isDirectory(path)) {
             String configFolder = Config.getConfigLocation();
             log.debug("Creating config folder {}", configFolder);
```

Comment:
LGTM! Just a tiny suggestion: I guess we could just print `path.toString()` here and erase line 88? WDYT?

Reply:
Yeah, it's basically getting `Config.getConfigLocation()` twice, once as a `Path` and then again as a `String`.